modules/ROOT/pages/anypoint-mq/anypoint-mq-connector-reference.adoc

```diff
@@ -189,9 +202,9 @@ Executes a Not Acknowledgment over a given AnypointMQMessageContext and changes
 | Output Encoding a| String |  The encoding of the payload that this operation outputs. |  |
 | Streaming Strategy a| * <<repeatable-in-memory-stream>>
 * <<repeatable-file-store-stream>>
-* non-repeatable-stream |  Configure to use repeatable streams. |  |
-| Target Variable a| String |  The name of a variable in which the operation's output is placed. |  |
-| Target Value a| String |  An expression to evaluate against the operation's output and the outcome of that expression is stored in the target variable. |  #[payload] |
+* non-repeatable-stream |  Configure if repeatable streams should be used and their behaviour |  |
```

Comment:
and their behavior...?

Is this supposed to convey something like:
Configure if repeatable strings and their behavior should be used. 

?